scout/server/blueprints/cases/views.py

```diff
@@ -439,7 +387,13 @@ def phenotypes_actions(institute_id, case_name):
             store.remove_phenotype(institute_obj, case_obj, user_obj, case_url, hpo_id)
 
     if action == "ADDGENE":
-        hgnc_ids = parse_raw_gene_ids(request.form.getlist("genes"))
+        try:
+            hgnc_ids = parse_raw_gene_ids(request.form.getlist("genes"))
+        except ValueError:
+            flash(
+                "Provided gene info could not be parsed! " "Please allow autocompletion to finish.",
```

Comment:
I don't trust users 🙂

I've made this crash, but can't reproduce it a work. Maybe I could do it on my home machine because it is so slow and unresponsive on some webpages.